### PR TITLE
add flammkuchen

### PR DESCRIPTION
**Pull request by hadim**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by hadim** [comment]
@conda-forge/help-python please review and merge. Thank you :-D

File: recipes/flammkuchen/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e9aab9b229ace70d879b85618a9ce0e88dd6ce35d4dbcdfd60c6b61c33a1b4fb
+
+build:
+  number: 0
+  skip: true  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (npavlovikj) [inline recipes/flammkuchen/meta.yaml]: This recipe can be `noarch`, so please remove the `skip` line and add `noarch: python` instead, https://conda-forge.readthedocs.io/en/latest/meta.html#building-noarch-packages.

File: recipes/flammkuchen/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (npavlovikj) [inline recipes/flammkuchen/meta.yaml]: With `noarch` use `python >=3.5` instead.

File: recipes/flammkuchen/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+    - numpy
+    - scipy
+    - pytables
+  run:
+    - python
```
Comment (npavlovikj) [inline recipes/flammkuchen/meta.yaml]: With `noarch` use `python >=3.5` instead.

File: recipes/flammkuchen/meta.yaml (hunk 2 above)
Comment (npavlovikj) [inline recipes/flammkuchen/meta.yaml]: For pure packages, usually only `python` and `pip` should be in the `host` section, so please remove the other dependencies.

**Comment by hadim** [comment]
Thanks @npavlovikj

**Comment by hadim** [comment]
Good to merge.

**Comment by npavlovikj** [comment]
Thank you for contributing @hadim 🎉

**Pull request merged**